x-pack/plugins/ml/public/application/settings/calendars/edit/calendar_form/calendar_form.tsx

```diff
@@ -193,38 +242,62 @@ export const CalendarForm = ({
           >
             <EuiComboBox
               onCreateOption={onCreateGroupOption}
-              options={groupIds}
+              options={groupIdOptions}
               selectedOptions={selectedGroupOptions}
               onChange={onGroupSelection}
               isDisabled={saving === true || canCreateCalendar === false || loading === true}
               data-test-subj="mlCalendarJobGroupSelection"
             />
           </EuiFormRow>
         </>
-      )}
+      ) : null}
 
       <EuiSpacer size="xl" />
 
       <EuiFormRow
         label={
-          <FormattedMessage
-            id="xpack.ml.calendarsEdit.calendarForm.eventsLabel"
-            defaultMessage="Events"
-          />
+          isDst ? (
```

Comment:
This should be disabled for a user without permission to edit DST calendars.

Reply:
Updated in https://github.com/elastic/kibana/pull/193605/commits/5b1a167d52cd3246688259ac3a26629b899709d8